src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -62,6 +62,45 @@ def forceCleanupWS() {
     run_shell(cmd, "Clean workspace")
 }
 
+Map getArchConf(config, arch) {
+    def k8sArchConfTable = getConfigVal(config, ['kubernetes', 'arch_table'], [:])
+
+    if (!k8sArchConfTable) {
```

Comment:
how it can be null? the last param on line 66 is empty hashtable. maybe it should be as:
```
if (k8sArchConfTable.size() ==0) {
```

Reply:
`[:]` is handled as false by groovy.
http://groovy-lang.org/semantics.html#Groovy-Truth